lib/meilisearch-rails.rb

```diff
@@ -280,8 +280,6 @@ def settings(*args)
       end
     end
 
-    private
-
     def self.log_or_throw(method, raise_on_failure, &block)
```

Comment:
The `log_or_throw` currently is being used as a public method in other places. I removed the `private` keyword because on ruby they don’t make any difference on static/class (self.) methods.